ibis/pyspark/compiler.py

```diff
@@ -1035,6 +1022,22 @@ def compile_join(t, expr, scope, timecontext, *, how):
     return left_df.join(right_df, pred_columns, how)
 
 
+def _canonicalize_interval(interval):
+    """ Convert interval to integer timestamp of second
```

Comment:
Perhaps comment here that when pyspark cast timestamp to integer type, it uses the number of seconds since epoch.

Reply:
Thanks, I added this to docstring.